ui-core/src/components/inputs/TimePicker.tsx

```diff
@@ -32,7 +42,14 @@ const TimeRange = ({ range, selectedItem, className, onSelectItem }: TimeRangePr
   </div>
 );
 
-const TimePicker = ({ onTimeChange, hours = 0, minutes = 0, ...otherProps }: TimePickerProps) => {
+const TimePicker = ({
+  onTimeChange,
+  hours = 0,
+  minutes = 0,
+  seconds = 0,
+  displaySeconds,
```

Comment:
Can you make displaySeconds false by default ?
```suggestion
  displaySeconds = false,
```

Reply:
yes, done